robotframework_reportportal/variables.py

```diff
@@ -62,6 +62,8 @@ def __init__(self):
             'RP_SKIPPED_ISSUE', default='True'))
         self.test_attributes = get_variable(
             'RP_TEST_ATTRIBUTES', default='').split()
+        self.verify_ssl = strtobool(get_variable(
```

Comment:
According to documentation `verify` parameters in `requests` library may also be a `string` with a path to a certificate: https://docs.python-requests.org/en/master/user/advanced/#ssl-cert-verification
So this will fail in that case.

Please update your code to be compatible with strings.